for_dict_challenges_bonus.py

```diff
@@ -65,6 +65,92 @@ def generate_chat_history():
         })
     return messages
 
+# 1. Вывести айди пользователя, который написал больше всех сообщений.
+def user_wrote_most_messages(history):
+    
+    user_id_message = {}  # создаётся словарь с парой - 'user_id': кол-во сообщений
+    for user_id in history:
+        user_id_message[user_id['sent_by']] = user_id_message.get(user_id['sent_by'], 0) + 1
+
+    user_id_max = max(user_id_message.items(), key=lambda items: items[1])  # определяю максимальное количество сообщений
+    
+    users_id_max = [] 
+    for key, value in user_id_message.items():  # для определения всех айди пользователей, которые написали больше всех сообщений
+        if value >= user_id_max[1]:
```

Comment:
```suggestion
        if value == user_id_max[1]:
```
Мы не ожидаем что там окажется что-то большее.